### PR TITLE
Add wcf container test program

### PR DESCRIPTION
**Pull request by yujayee**
#21 
@Lxiamail

### CODE REVIEW

File: WcfContainerTest/WcfClient/App.config [hunk 1]
```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (zhenlan) [inline WcfContainerTest/WcfClient/App.config]: We are in the wcf-docker repo. We don't have to say this is WcfContainerTest. We can just add a "tests" (lower case) folder.

**Comment by zhenlan** [comment]
@yujayee can you please explain steps to run these tests?

Add @KKhurin for code review.

**Comment by zhenlan** [comment]
"WcfContainerTest/WcfServiceWebApp/webtest - Shortcut.lnk" should be excluded.

**Comment by yujayee** [comment]
Steps:
1. Copy WcfServiceWebApp folder (or WcfServiceConsoleApp folder, depends on if we are testing wcf image or dotnet-framework image) to the server machine
2. The copied folder contains Dockerfile, build and run the container on server.
3. Copy WcfClient and packages folders to client machine.
4. Set environment variable "host" to IP address of server machine, create a log path, run the .exe in WcfClient and pass in the log path as parameter
5. Verify if the exit code is 0.
@zhenlan @KKhurin

**Comment by Lxiamail** [comment]
Adding the steps as a readme file under tests folder may be helpful. Also the steps may contain all detailed steps, like build the project, how to set the "image" variable, logpath for client app, etc.

**Comment by yujayee** [comment]
@Lxiamail I not sure if adding a readme file is helpful. Our test is integrated into IIS system and the detailed functions like how to build and run container is defined in the powershell scripts in their system. The programs here in this repo are just test programs that are going to be copied to their designated folders and run. In other words, these are just test content, the commands on how to run these tests are not here.

**Comment by yujayee** [comment]
Close this pr since current test approach needs to be changed.

**Pull request closed without merging**